src/frontends/tensorflow/src/op/avg_pool.cpp

```diff
@@ -14,61 +14,43 @@ namespace tensorflow {
 namespace op {
 
 OutputVector translate_avg_pool_op(const NodeContext& node) {
-    Output<Node> ng_input = node.get_input(0);
+    TENSORFLOW_OP_VALIDATION(node, node.get_input_size() > 0, "AvgPool and AvgPool3D must have at least one input.");
+    auto input = node.get_input(0);
 
+    auto spatial_dim = (node.get_op_type() == "AvgPool") ? 2 : 3;
```

Comment:
Could you add check that only AvgPool and AvgPool3D is a valid options.

Reply:
Done